src/main/java/ui/components/pickers/MilestonePickerState.java

```diff
@@ -12,14 +13,13 @@
  * which determine the result of the user's input
  */
 public class MilestonePickerState {
-    private List<PickerMilestone> currentMilestonesList;
-
-    public MilestonePickerState(List<PickerMilestone> milestones) {
-        currentMilestonesList = cloneList(milestones);
-    }
```

Comment:
It won't be used anymore since the initialisation will pass the list of milestones together with an empty string instead.